Classes/View/PdfView.php

```diff
@@ -127,7 +127,21 @@ private function replaceStrings($content) {
             }
         }
 
-        return $content;
+        return preg_replace_callback('/\<a (.*?)>/', function ($hit) {
+
+            $completeATag = $hit[0];
+            $returnTag = preg_replace_callback('/href="(.*)#(.*)"/', function ($href) {
+
+                if (!empty($href[2]) &&
+                        ($href[1] === '/' . str_replace('&', '&amp;', GeneralUtility::getIndpEnv(TYPO3_SITE_SCRIPT)))
```

Comment:
1. Shouldn't `TYPO3_SITE_SCRIPT` be quoted? Otherwise, you'll call `getIndpEnv()` with the value of that constant.
2. Use `htmlentities` instead of `str_replace`?